TombEditor/SectorsClipboardData.cs

```diff
@@ -36,19 +36,26 @@ public SectorsClipboardData(Editor editor)
                             var currZ = selection.Area.Y0 + z;
                             var b = editor.SelectedRoom.Blocks[currX, currZ];
 
-                            for (BlockVertical vertical = 0; vertical < BlockVertical.Count; ++vertical)
+                            BlockVertical[] verticals = b.GetVerticals().ToArray();
+                            writer.Write(verticals.Length);
+
+                            foreach (BlockVertical vertical in verticals)
+                            {
+                                writer.Write((byte)vertical);
+
                                 for (BlockEdge edge = 0; edge < BlockEdge.Count; ++edge)
-                                    writer.Write((short)b.GetHeight(vertical, edge));
+                                    writer.Write(b.GetHeight(vertical, edge));
+                            }
 
-                            writer.Write((int)b.Type);
```

Comment:
Why `int` was converted to `byte` here?

Reply:
The `.Type` property is an `enum : byte`, so why on earth would you want to cast it to `int`? It's costly.